tests/v2/validation/workloads/workload_test.go

```diff
@@ -55,6 +55,20 @@ func (w *WorkloadTestSuite) TestWorkloadDeployment() {
 	require.NoError(w.T(), err)
 }
 
+func (w *WorkloadTestSuite) TestWorkloadSideKick() {
+	subSession := w.session.NewSession()
+	defer subSession.Cleanup()
+
+	_, namespace, err := projectsapi.CreateProjectAndNamespace(w.client, w.cluster.ID)
+	require.NoError(w.T(), err)
+
+	_, err = deployment.CreateDeployment(w.client, w.cluster.ID, namespace.Name, 1, "", "", false, false)
+	require.NoError(w.T(), err)
+
+	_, err = deployment.CreateDeployment(w.client, w.cluster.ID, namespace.Name, 1, "", "", false, false)
```

Comment:
For the side kick test, we should not be creating a new deployment instead we create a single deployment with 2 containers. In the original python test we are updating an existing deployment and add a new container in that deployment. 

Could you update this test and add a new container instead of creating a new deployment?

https://github.com/rancher/rancher/blob/main/tests/validation/tests/v3_api/test_workload.py#L39

Reply:
I got it, yes, I can

Reply:
Hi @anupama2501, done!